packages/kn-plugin-workflow/.vscode/tasks.json

```
@@ -21,8 +21,25 @@
       "type": "shell",
       "group": "none",
       "problemMatcher": [],
-      "label": "clean image",
-      "command": "docker image rm kn-workflow:debug"
+      "label": "create",
+      "command": "kn workflow create --name='debug'"
+    },
+    {
+      "type": "shell",
+      "group": "none",
+      "problemMatcher": [],
+      "label": "quarkus create",
+      "command": "kn workflow quarkus create --name='debug'"
+    },
+    {
+      "label": "build and create",
+      "dependsOrder": "sequence",
+      "dependsOn": ["build", "create"]
+    },
+    {
+      "label": "quarkus build and create",
```

I suggest using `build and quarkus create` instead, to avoid misinterpretation, since it builds the binary and then performs `quarkus create`. Maybe even replace `build` with `buildBinary` to make it explicit that it is not the `kn workflow quarkus build`.
```suggestion
      "label": "build and quarkus create",
```

---

done